app/views/layout.erb

```diff
@@ -15,13 +15,24 @@
 <body>
   <div class="container">
     <strong>Together, we can TODO it!</strong>
+    <% if !session[:email]%>
```

Comment:
Having a `current_user` helper would be an improvement on looking directly at the session here.